controllers/apply/lib/field-types/file.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+const mime = require('mime-types');
+const filesize = require('filesize');
+
+const Field = require('./field');
+const Joi = require('../joi-extensions');
+
+class FileField extends Field {
+    constructor(props) {
+        super(props);
+
+        this.labelExisting = props.labelExisting || null;
+
+        const maxFileSize = {
+            value: 12 * 1048576,
+            label: '12MB'
+        };
+
+        const supportedFileTypes = [
```

Comment:
Likewise, made supported file types static as we only have one type of file upload so not too worried about this being configurable at the moment.